ledger-tool/src/main.rs

```diff
@@ -1684,6 +1701,37 @@ fn main() {
                 .global(true)
                 .help("Use DIR for separate incremental snapshot location"),
         )
+        .arg(
+            Arg::with_name("replaying_backend")
+                .long("replaying-backend")
+                .value_name("BACKEND")
+                .takes_value(true)
+                .possible_values(&[
```

Comment:
I guess this protects us from panicking above like I mentioned.

I'm not a huge fan of all stringified variants of the enums being duplicated here; maybe we should use something like `strum::EnumIter`? (https://docs.rs/strum/latest/strum/derive.EnumIter.html)

Reply:
fixed: b9d517cc752aa0d19dfb9df3c69190bca68f1646

btw, i got fond of `strum`. :) it solved all itching points at once. I wonder why no one used for clap+enum case in our monorepo...